docs/guides/tutorials/trpc.rst

```diff
@@ -0,0 +1,468 @@
+.. _ref_guide_trpc:
+
+Integrating EdgeDB with tRPC
+============================
+
+This guide explains how to integrate **EdgeDB** with **tRPC** for a modern, 
+type-safe API. We'll cover setting up database interactions, API routing, 
+and implementing authentication, all while ensuring type safety across the 
+client and server.
+
+You can reference the following repositories for more context:
+
+- `TurboRepo Template <https://github.com/scotttrinh/turbo-edb-app/blob/main-template/>`_: A monorepo template using **TurboRepo**, **EdgeDB**, and **tRPC**.
+- `LookFeel Project <https://github.com/LewTrn/lookfeel>`_: A real-world project example using **EdgeDB** and **tRPC**.
+
+Step 1: EdgeDB setup
+====================
+
+EdgeDB will serve as the database layer for your application.
+
+Install and initialize EdgeDB
+-----------------------------
+
+To initialize **EdgeDB**, run the following command using your preferred 
+package manager:
+
+.. code-block:: bash
+
+   $ pnpm dlx edgedb project init # or npx edgedb project init
+
+This will create an EdgeDB project and set up a schema to start with.
+
+Define the EdgeDB Schema
+------------------------
+
+The previous command generated a schema file in the `dbschema` directory.
+
+Here’s an example schema that defines a `User` model:
+
+.. code-block:: sdl
```

Comment:
Maybe add `:caption:` blocks to these files to make it a little more concrete what the structure is?

```suggestion
.. code-block:: sdl
   :caption: dbschema/default.esdl
```